pom.xml

```diff
@@ -33,6 +33,7 @@
     <lombok.version>1.18.16</lombok.version>
     <spring.version>5.2.7.RELEASE</spring.version>
     <maven.site.plugin.version>3.9.1</maven.site.plugin.version>
+    <sonar.exclusions>src/main/java/org/folio/circulation/domain/LoanAndRelatedRecords.java</sonar.exclusions>
```

Comment:
Why has this been excluded from Sonar's checking?

Reply:
I removed this exclusion. The problem with code coverage comes from using jmockit javaagent. It is automatically fixed when I deleted jmockit.